hack/bundle-gen.py

```diff
@@ -146,7 +146,7 @@ def build_and_push_image(registry_auth_file, hive_version, dry_run):
     if registry_auth_file != None:
         cmd = cmd + " --authfile={} ".format(registry_auth_file)
     cmd = cmd + " {}".format(container_name)
-    subprocess.run(cmd.split())
+    subprocess.run(cmd.split(), check=True)
```

Comment:
We should do this for building the image as well but I expect that will succeed most of the time.

Reply:
@abutcher i went ahead an added it to the build subprocess as well.